src/main/java/com/github/gokolo/personaldashboard/servlet/ListUsersServlet.java

```diff
@@ -14,26 +13,21 @@
 import com.github.gokolo.personaldashboard.dao.UserDAOImpl;
 import com.github.gokolo.personaldashboard.dto.UserDTO;
 
+/**
+ * @author Grace_Okolo
+ *
+ */
 @SuppressWarnings("PMD")
-@WebServlet(name = "ListUsersServlet", urlPatterns = "/xxx")
+@WebServlet(name = "ListUsersServlet", urlPatterns = "/users")
 public class ListUsersServlet extends HttpServlet {
     private UserDAO userDAO = new UserDAOImpl();
 
     public void doGet(final HttpServletRequest request, final HttpServletResponse response)
             throws IOException, ServletException {
         response.setContentType("text/html");
         List<UserDTO> userCollection = userDAO.findAll();
-        StringBuilder sb = new StringBuilder();
-        sb.append("<!DOCTYPE html>").append("<html>").append("<head>").append("<title>").append("All Users")
-                .append("</title>").append("</head>").append("<body>");
-
-        for (UserDTO user : userCollection) {
-            sb.append("<p>").append(user.getName()).append("</p>");
-        }
-        sb.append("</body>").append("</html>");
-
-        PrintWriter writer = response.getWriter();
-        writer.println(sb.toString());
+        request.setAttribute("user", userCollection);
```

Comment:
Misleading name for attribute.
The object is a collection of `UserDTO` objects so calling them `user` can confuse your colleagues and can make your code harder to understand.